manifests/init.pp

```diff
@@ -259,11 +259,12 @@
     # parameter values from the main ::certs class.  Kafo can't handle that case, so
     # it remains here for now.
     include ::puppet
-    include ::puppet::server
```

Comment:
Why remove this?

Reply:
This was silently pulling in the puppet server which users might not want. The main puppet class has a parameter whether to include it and also correct chaining. You might be right that there are cases where a user has manually included it and this isn't correct. Coupling to other modules is annoying.

Reply:
@sean797 do you think I should add a `fail()` instead when those variables are false?

Reply:
What do you man by `it` here? The include or a `fail()` message?

Reply:
sorry the include, so

```
if $::puppet::server and $::puppet::server::foreman {
  include ::puppet::server
```

Reply:
https://github.com/theforeman/puppet-puppet/blob/837e65929a7ba12e2bc730ec7e817c33acda3f57/manifests/init.pp#L743-L746 should already take care of that

Reply:
in that case, I'm okay with this :+1: